proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -6,10 +6,13 @@ import "google/protobuf/wrappers.proto";
 
 option go_package = "github.com/Traceableai/agent-config/gen/go/v1";
 
-// Parent configuration object for Opa & Blocking Config
 message AgentConfig {
-    Opa opa = 1;
-    BlockingConfig blocking_config = 2;
+  Opa opa = 1;
+  BlockingConfig blocking_config = 2;
+  google.protobuf.BoolValue debug_log = 3;
```

Comment:
@davexroth the ugliest is this `debug_log`, any idea to make it better? This turns on libtraceable stdout logging.

Reply:
yeah, this one sucks.  Can you look at our docs?  Is it documented anywhere or in any of our shipped examples?  If not, I think we can remove it, but if it is, we'll just have to live with it. I suspect if it's referenced anywhere, it'll be in nginx.

Reply:
yep

Reply:
oh, wait!  i'm an idiot - this `debug_log` is new right?  So we can call it whatever we want.  it's only the existing one in `BlockingConfig` we shouldn't change if it's been documented anywhere

Reply:
It is! I just don't know what's a good name.

Reply:
`debug`? or I'm also ok with just leaving as `debug_log` too

Reply:
Leaving as `debug_log`. We should be good to go with the PR, all I've done with existing messages are adding deprecated tags.